201-virtual-wan-with-all-gateways/deploy_vwan.ps1

```diff
@@ -0,0 +1,77 @@
+﻿# Deploy vWAN environment using ARM template and parameter file #
```

Comment:
remove this file - all samples in the repo must be deployable with the scripts in the repo